src/components/ReviewChats/MyChatsFilterer.tsx

```diff
@@ -45,8 +48,15 @@ const DetailsModalButton = ({ chatRequest }: DetailsModalButtonProps) => {
           setIsOpen(false);
         }}
       >
+        <ProfileModal
+          isOpen={profileModalOpen}
+          onClose={() => {
+            setProfileModalOpen(false);
+          }}
+          profile={chatRequest.mentorProfile as Profile}
```

Comment:
Why is this cast needed?

Reply:
It just throws a type error if you don't cast it : /